tutorial/consuming_packages/use_tools_as_conan_packages.rst

```diff
@@ -48,6 +48,18 @@ using CMake **v3.22.6**.
     CMakeDeps
     CMakeToolchain
 
+
+.. important::
+
+    Please note that this *conanfile.txt* will install *zlib/1.2.11* and *cmake/3.22.6*
+    separately. However, if Conan does not find a binary for Zlib in Conan Center and it
+    needs to be built from sources, a CMake installation must already be present in your
+    system. If you want to use that *cmake/3.22.6* to also build Zlib, when installing if
+    necessary, you may add the ``[tool_requires]`` section to the profile you are using.
+    Please check :ref:`the profile doc<reference_config_files_profiles>` for more
+    information.
```

Comment:
```suggestion
    Please note that this *conanfile.txt* will install *zlib/1.2.11* and *cmake/3.22.6*
    separately. However, if Conan does not find a binary for Zlib in Conan Center and it
    needs to be built from sources, a CMake installation must already be present in your
    system, because the ``cmake/3.22.6`` declared in your ``conanfile.txt`` only applies to 
    your current project, not all dependencies.
    If you want to use that *cmake/3.22.6* to also build Zlib, when installing if
    necessary, you may add the ``[tool_requires]`` section to the profile you are using.
    Please check :ref:`the profile doc<reference_config_files_profiles>` for more
    information.
```